extensions/dice-and-coin/src/throw-dice.tsx

```diff
@@ -1,7 +1,7 @@
 import { showToast, Toast } from "@raycast/api";
 
 export default async function Command() {
-  const toast = await showToast({ style: Toast.Style.Animated, title: "Throwing die..." });
```

Comment:
@pernielsentikaer but die is the singular of dice https://en.wiktionary.org/wiki/die#Noun_2

Reply:
Let's die, sorry 😊